Mexico Wall Update

VirenMohindra opened the pull request:
http://www.reuters.com/article/us-usa-trump-immigration-idUSKBN1591HP

VirenMohindra committed:
update of mexico wall

http://www.reuters.com/article/us-usa-trump-immigration-idUSKBN1591HP

VirenMohindra commented:
Either we can squash all of these promises together, or update each one separately.

@MrLuit @alanfriedman

MrLuit commented:
I suggest that we make one promise for all the 'build a wall' policies but include all necessary information

VirenMohindra committed:
combined duplicates, gave importance to 100 days

combined policies for trump wall

- ‘Deport all undocumented immigrants. Trump''s campaign began with a
promise to build a wall across the United States'' southern border and
deport the country''s 11 million undocumented immigrants.'

- '"The End Illegal Immigration act fully funds the construction of a
wall on our southern border," Trump said on October 22nd, outlining the
plans for his first 100 days in office. "With the full understanding
that the country of Mexico will be reimbursing the United States for
the full cost of such a wall, OK?"'
